app/src/main/java/de/danoeh/antennapod/ui/episodeslist/FeedItemMenuHandler.java

```diff
@@ -54,7 +54,8 @@ private FeedItemMenuHandler() {
      * @param excludeIds Menu item that should be excluded
      * @return Returns true if selectedItem is not null.
      */
-    public static boolean onPrepareMenu(Menu menu, List<FeedItem> selectedItems, int... excludeIds) {
+    public static boolean onPrepareMenu(Menu menu, List<FeedItem> selectedItems, List<FeedItem> queue,
+                                        int... excludeIds) {
```

Comment:
Hmm, passing the queue feels a bit hacky. I don't have a better solution but I think this doesn't really belong here.